ingress/ingress-nginx-dep.yaml

```diff
@@ -39,14 +49,14 @@ spec:
                   fieldPath: metadata.namespace
           resources:
               requests:
-                memory: "256Mi"
+                memory: "128Mi"
               limits:
-                memory: "256Mi"
+                memory: "128Mi"
           ports:
-          - name: http
-            containerPort: 80
-          - name: https
-            containerPort: 443
+            - name: http
```

Comment:
Should this be limited to https only?

Reply:
I've added an issue to discuss this. https://github.com/qlik-oss/core-scaling/issues/89